.eslintrc.js

```diff
@@ -18,7 +18,6 @@ module.exports = {
 		'plugin:functional/external-recommended', // https://github.com/jonaskello/eslint-plugin-functional
 		'plugin:functional/recommended', // https://github.com/jonaskello/eslint-plugin-functional
 		'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
-		'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
```

Comment:
reasoning?

Reply:
had some conflicts between my local prettier config and eslint prettier. Disabled it during development and accidentally pushed it